src/Microsoft.DotNet.Build.Tasks/SaveItems.cs

```diff
@@ -0,0 +1,57 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Build.Framework;
+using Microsoft.Build.Construction;
+using System.Collections.Generic;
+using System.Linq;
+using System.IO;
+
+namespace Microsoft.DotNet.Build.Tasks
+{
+    public class SaveItems : BuildTask
+    {
+        [Required]
+        public string ItemName { get; set; }
+
+        [Required]
+        public ITaskItem[] Items { get; set; }
+
+        [Required]
```

Comment:
I added output to this with https://github.com/dotnet/corefx/commit/7389e08300d9bab19e2604e61a30bf0720444c7e